docs/serverless/ingest/auto-import.mdx

```diff
@@ -0,0 +1,87 @@
+---
+slug: /serverless/security/automatic-import
+title: Automatic Import
+description: Use Automatic Import to quickly ingest third-party data.
+tags: [ 'serverless', 'security', 'how-to' ]
+status: in review
+---
+
+<DocBadge template="technical preview" />
+
+<DocCallOut title="Technical preview" color="warning">
+This feature is in technical preview. It may change in the future, and you should exercise caution when using it in production environments. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of GA features.
+</DocCallOut>
+
+Automatic Import helps you quickly parse, ingest, and create [ECS mappings](https://www.elastic.co/elasticsearch/common-schema) for data from sources that don't yet have prebuilt Elastic integrations. This can accelerate your migration to ((elastic-sec)), and help you quickly add new data sources to an existing SIEM solution in ((elastic-sec)). Automatic Import uses a large language model (LLM) with specialized instructions to quickly analyze your source data and create a custom integration. 
+
+While Elastic has 400+ [prebuilt data integrations](((integrations-docs))), Automatic Import helps you extend data coverage to other security-relevant technologies and applications. Elastic integrations (including those created by Automatic Import) normalize data to [the Elastic Common Schema (ECS)](https://www.elastic.co/guide/en/ecs/current/ecs-reference.html), which creates uniformity across dashboards, search, alerts, machine learning, and more. 
+
+
+<DocCallOut title="Tip">
+Click [here](https://elastic.navattic.com/automatic-import) to watch a demo of how Automatic Import works before setting it up yourself.
+</DocCallOut>
+
+<DocCallOut title="Requirements">
+
+- A working <DocLink slug="/serverless/security/connect-to-bedrock" text="Amazon Bedrock connector"/>. Automatic Import currently works with all variants of Claude 3. In this technical preview, other models are not yet supported.
+- A [Security Analytics Complete](https://www.elastic.co/pricing/serverless-security) subscription.
+- A sample of the data you want to import in JSON or NDJSON format. 
```

Comment:
```suggestion
- A sample of the data you want to import, in JSON or NDJSON format. 
```

Reply:
I think I had a comma here before Janeen suggested I remove it in her review! :lol: